src/Http/Http.Extensions/src/HttpResponseJsonExtensions.cs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+#nullable enable
+
+namespace Microsoft.AspNetCore.Http.Json
+{
+    public static partial class HttpResponseJsonExtensions
```

Comment:
I think we should make these `Task` instead since the underlying serialize returns `Task`